contracts/InternalMarket/InternalMarketBase.sol

```diff
@@ -221,10 +223,12 @@ contract InternalMarketBase {
 
     function _deposit(address to, uint256 amount) internal virtual {
         tokenInternal.wrap(to, amount);
+        emit Deposited(to, amount);
```

Comment:
I have this handler in subgraph, in agreement with @vrde :
```ts
export function handleDepositStarted(event: DepositStarted): void {
  const from = event.params.from;
  const amount = event.params.amount;
  const settlementTimestamp = event.params.settlementTimestamp;

  const id = from.toHexString() + "-" + settlementTimestamp.toString();
  const deposit = new Deposit(id);

  deposit.from = from;
  deposit.amount = amount;
  deposit.settlementTimestamp = settlementTimestamp;
  deposit.save();
}
```
So, I'm not sure we will need this event at all?